content/pages/comparisons/index.html

```diff
@@ -0,0 +1,132 @@
+{# View this page at http://localhost:8080/comparisons/ #}
+
+<html>
+  <head>
+    <title>Read the Docs vs Other Documentation Platforms</title>
+    <meta name="description" content="Compare Read the Docs features with other documentation platforms like GitHub Pages, GitBook, and Cloudflare Pages to find the best solution for your documentation needs." />
+    <meta name="template" content="page" />
+    <meta name="slug" content="comparisons" />
+    <meta name="status" content="hidden" />
+  </head>
+  <body>
+
+  {% import "partials/basic.html" as basic %}
+
+  <section>
+    {% call basic.header_lead(
+    header="Compare Read the Docs with other platforms")
+    %}
+    <p>
+      While there are many options for hosting documentation,
+      Read the Docs is purpose-built for technical documentation with features that make both documentation authors and readers more productive.
+    </p>
+    <p>
+      Compare Read the Docs with other popular documentation platforms to find the best solution for your needs.
+    </p>
+    {% endcall %}
+  </section>
+
+  <section>
+    <div class="ui very padded container">
+      <div class="ui three stackable cards">
+
+        <div class="card">
```

Comment:
Can we make each of these cards linkable instead? Does that make sense?

Reply:
I tried a couple approaches, and hit random annoying things with css. But I can try that.